### PR TITLE
Adds missed publications and new ones for January 2023

### PR DESCRIPTION
**Pull request by ljhwang**
Adds new publications for January 2023 and a few past missed ones - thanks to everyone for pasting your doi's into the chat today.

Also updates the publication information for a previous publication.

### CODE REVIEW

**Comment by trajaona** [comment]
Hi Lorraine, Sorry but I just noticed that this publication is missing from the list:
T.A. Rajaonarison, D.S. Stamps, J. Naliboff (2021), Role of Lithospheric Buoyancy Forces in Driving Deformation in East Africa, Geophysical Research Letters. doi: [10.1029/2020GL090483](https://doi.org/10.1029/2020GL090483).

File: doc/manual/citing_aspect.bib [hunk 1]
```diff
@@ -1450,3 +1451,88 @@ @article{LIU2022105509
...
+}
+ticle{https://doi.org/10.1029/2022GL099286,
```
Comment (tjhei) [inline doc/manual/citing_aspect.bib:1492]: This seems to be broken!

Comment (ljhwang) [reply doc/manual/citing_aspect.bib:1492]: it does mystify me how these things happen.

**Comment by tjhei** [comment]
We need to be careful when merging this as #5049 is more strict with bib checking. Ideally, we would merge that one first, rebase this one, and check for errors...

**Comment by tjhei** [comment]
@ljhwang Would you be able to rebase to the current ASPECT master? Let me know if you need help with that.

**Comment by ljhwang** [comment]
Well, you know, I would not mind some expert help :)

**Comment by tjhei** [comment]
> Well, you know, I would not mind some expert help :)

assuming that the official repository is called "upstream" on your machine (you can check with ``git remote -v``):
```
git remote update
git rebase upstream main
```

Then push the result (you will need ``--force``).

I just checked and there is one new warning that you will need to fix:
```
Biber warning: [280] Utils.pm:293> WARN - Duplicate entry key: 'Bahadori2022' in file './manual/citing_aspect.bib', 
```

**Comment by ljhwang** [comment]
Thanks for the git tips @tjhei

File: doc/manual/citing_aspect.bib [hunk 2]
```diff
@@ -1450,3 +1451,88 @@ @article{LIU2022105509
...
+and Austermann, Jacqueline
+and Campbell, Lajhon
+and Rasbury, E. Troy
+and Davis, Daniel M.
+and Calvelage, Christopher M.
+and Flesch, Lucy M.},
+title={The role of gravitational body forces in the development of metamorphic core complexes},
+journal={Nature Communications},
+year={2022},
+month={Sep},
```
Comment (tjhei) [inline doc/manual/citing_aspect.bib:1467]:
```suggestion
month=sep,
```

**Review by tjhei — APPROVED** [review]

**Pull request merged**